content/tutorials/manage-the-rippled-server/troubleshooting/understanding-log-messages.md

```diff
@@ -42,6 +42,24 @@ Occasional messages of this type do not usually indicate a problem. If this type
 - The validator described in the message is behaving maliciously.
 
 
+## async_send failed
+
+The following log message indicates that [StatsD export](configure-statsd.html) failed:
+
+```text
+2020-Apr-07 16:59:36.299867439 UTC Collector:ERR async_send failed: Connection refused
+```
+
+This could mean:
+
+- Your StatsD configuration has the wrong IP address or port.
+- The StatsD server you are exporting to is down or not accessible from your `rippled` server.
```

Comment:
Arbitrary, but I suggest
The StatsD server you are exporting to  -> The StatsD server that you attempted to export to

Reply:
As mentioned above, this export is a continuous log of analytics, so in this case I think the progressive tense is a better fit than the past tense. Though, I agree it could be revised to account for the fact that it failed.

Revised accordingly. :+1: